chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
-and allows you to recover those chunks easily
-if you ever need to get back previous work.
-Therefore, version control supports a tidy workspace.
-(It also allows effective collaboration in this space,
-but that is a more advanced tool not covered here.)
-
-Organizing your workspace effectively is now essential.
-\texttt{iefolder} will have created a \texttt{/Dofiles/Analysis/} folder;
-you should feel free to add additional subfolders as needed.
-It is completely acceptable to have folders for each task,
-and compartmentalize each analysis as much as needed.
-It is always better to have more code files open
-than to have to scroll around repeatedly inside a given file.
-Just like you named each of the analysis datasets,
-each of the individual analysis files should be descriptively named.
-Code files such as \path{spatial-diff-in-diff.do},
-\path{matching-villages.do}, and \path{summary-statistics.do}
-are clear indicators of what each file is doing
-and allow code to be found very quickly.
-Eventually, you will need to compile a ``release package''
-that links the outputs more structurally to exhibits,
-but at this stage function is more important than structure.
-
-Outputs should, whenever possible, be created in lightweight formats.
-For graphics,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Checklist:_Reviewing_Graphs}} \texttt{.eps} or \texttt{.tif} files are acceptable;
-for tables,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Checklist:_Submit_Table}} \texttt{.tex} is preferred (don't worry about making it
-really nicely formatted until you are ready to publish).
-Excel \texttt{.xlsx} files are also acceptable,
-although if you are working on a large report
-they will become cumbersome to update after revisions.
-If you are writing your code in Git/GitHub,
-you should output results into that directory,
-not the directory where data is stored (i.e. Dropbox).
-Results will differ across different branches
-and will constantly be re-writing each other:
-you should have each result stored with the code that created it.
 
-%------------------------------------------------
+% Exploratory and final data analysis -----------------------------------------
+The analysis stage usually starts with a process we call exploratory data analysis.
+This is when you are trying different things and looking for patterns in your data. 
+It progresses into final analysis when your team starts to decide what are the main results, those that will make it into the research output.
+The way you deal with code and outputs for exploratory and final analysis is different, and this section will discuss how.
+% Organizing scripts ---------------------------------------------------------
+During exploratory data analysis, you will be tempted to write lots of analysis into one big, impressive, start-to-finish script. 
+Though it's fine to write such a script during a long analysis meeting, this practice is error-prone.  
```

Comment:
"_during a long analysis meeting_"  - we need to explain what we mean here if this is something we want to keep

Reply:
>Although it's fine to write such a script if you are coding in real-time during a long analysis meeting with your PIs, this practice is error-prone.  

(c80a095)

Reply:
I am not sure I get it still. Is this common that people write code in meetings? What is the point we are making here? That things does not have to be coded in the order they were discussed?

Then I also think it is a too explicit RA focus. I think we can say "_during a long analysis meeting with **the rest of your research team**_"

Reply:
I certainly do. But I am not very attached to this sentence.